shell.nix

```diff
@@ -7,13 +7,6 @@ let
     packageOverrides = p: rec {
       sbt = p.sbt.overrideAttrs (
         old: rec {
```

Comment:
The sbt version in nixos-21.05 is `1.5.4`. I don't think we need to override it.

Reply:
I get `1.5.1`?   Either way, it gets the job done.